roles/ansible-os-hardening/vars/sysctl.yml

```diff
@@ -83,7 +83,7 @@ sysctl_config:
   # This settings controls how the kernel behaves towards module changes at
   # runtime. Setting to 1 will disable module loading at runtime.
   # Setting it to 0 is actually never supported.
-  kernel.modules_disabled: 1
+  # kernel.modules_disabled: 1
```

Comment:
The tests specify that this value should be zero:
```
  context linux_kernel_parameter('kernel.modules_disabled') do
    its(:value) { should eq 0 }
  end
```
However setting this variable to 0 with Ansible results in an error, because 0 is not supported. That's why I commented it out. The user has to enable the setting by himself.